src/Stripe.net/Services/Accounts/AccountListOptions.cs

```diff
@@ -1,7 +1,14 @@
 // File generated from our OpenAPI spec
 namespace Stripe
 {
+    using System;
+    using Newtonsoft.Json;
+    using Stripe.Infrastructure;
+
     public class AccountListOptions : ListOptionsWithCreated
     {
+        [JsonProperty("created")]
+        [JsonConverter(typeof(AnyOfConverter))]
+        public AnyOf<DateTime?, DateRangeOptions> Created { get; set; }
```

Comment:
In .NET there is no "shadowing" of properties. You end up defining a separate property with the same name and it has bad side-effects.

Reply:
```C#
var options = new AccountListOptions();
options.Created = DateTime.Now;

Console.WriteLine(((ListOptionsWithCreated)options.Created); // prints null
```

Reply:
Gotcha! Thanks for the insight. I think it'd be breaking to simply remove this from the ListOptionsWithCreated class and move it into the children instead, am I understanding correctly?

Reply:
Yep, that's breaking as well.